/dart_config:
  ros__parameters:
    auto_fw_calibration: false
    auto_yaw_calibration: false
    dart_selection:
      - 7.34
      - 7.35
      - 7.27
      - 7.32
    target_delta_height: 0.240000
    target_distance: 25.500000
    target_fw_velocity: 5870
    target_fw_velocity_launch_offset:
      - 0
      - 0
      - 0
      - 0
    target_fw_velocity_offset: 0
    target_fw_velocity_ratio: 1.414000
    target_yaw_angle: 160736
    target_yaw_angle_offset: 0
    target_yaw_launch_angle_offset:
      - 0
      - 0
      - 0
      - 0
    target_yaw_x_axis: 780.000000